Resources/Prototypes/Entities/Objects/Fun/figurine_boxes.yml

```diff
@@ -133,3 +133,5 @@
       - id: ToyFigurineHamlet
         prob: 0.20
         orGroup: SpacemenFig
+    sound:
+      path: /Audio/Effects/unwrap.ogg
```

Comment:
This should already be inherited from PresentBase. Is it not?

Reply:
Doesn't the component get overwritten and the path reset to the default?

Reply:
Component fields actually get merged with inheritance, so it should be fine. 

The issue of wiping out parent data comes up when the field's data type is more complex and you try to change a value within it. Those don't get merged.

Like here, the `sound` field of SpawnItemsOnUse is a SoundSpecifier, which has a few data fields of its own, like the resource path and any audio parameters like volume or variance that should be applied. If you wanted to inherit from PresentBase but keep the same resource path and only change the volume, that wouldn't work - it would try to replace the entire SoundSpecifier as one field. The fields _within_ the SoundSpecifier wouldn't get merged.

Reply:
Oh, I did not know that. Thanks for the explanation. Give me one moment, I will change it back.